app/models/accessories.rb

```diff
@@ -0,0 +1,3 @@
+class Accessories < ApplicationRecord
```

Comment:
great code - my only thought is that this file and class should be called "Accessory" instead of "Accessories" to keep with the weird pluralization rails does.  But if the tests pass I'm happy.